src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/SoundCloudWidgetPageObject.java

```diff
@@ -11,14 +13,19 @@
 
   @FindBy(css = "iframe[data-wikia-widget=\"soundcloud\"]")
   private WebElement soundCloudIframe;
+  @FindBy(css = "iframe[data-wikia-widget=\"soundcloud\"]")
+  private List<WebElement> soundCloudIframeList;
   @FindBy(css = "div.widget")
   private WebElement soundCloudBody;
 
   private static final String TAG_NAME = "soundcloud";
   private static final String ARTICLE_NAME = "SoundCloudWidget";
-  private static final String TAG =
+  private static final String TAGS[] = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)